dfget/config/constants.go

```diff
@@ -53,6 +53,7 @@ const (
 	DefaultLocalLimit      = 20 * rate.MB
 	DefaultMinRate         = 64 * rate.KB
 	DefaultClientQueueSize = 6
+	DefaultTotalLimit      = 20 * rate.MB
```

Comment:
Put this line above

Reply:
OK

Reply:
You mean Line  DefaultTotalLimit      = 20 * rate.MB  is placed in front of line  DefaultClientQueueSize = 6  
such as:
DefaultTotalLimit      = 20 * rate.MB
DefaultClientQueueSize = 6

Reply:
Right below the line of locallimit